File: app/src/main/java/workshop1024/com/xproject/model/message/MessageGroup.java
package workshop1024.com.xproject.model.message;

import java.util.List;

public class MessageGroup {
    private String mGroupId;
    private String mPublishData;
    private List<Message> mMessageList;

    public MessageGroup(String groupId, String publishData, List<Message> messageList) {
        mGroupId = groupId;
        mPublishData = publishData;
        mMessageList = messageList;
    }

    public String getGroupId() {
        return mGroupId;
    }

    public void setGroupId(String groupId) {
        mGroupId = groupId;
    }

    public String getPublishData() {
        return mPublishData;
    }

    public void setPublishData(String publishData) {
        mPublishData = publishData;
    }

    public List<Message> getMessageList() {
        return mMessageList;
    }

    public void setMessageList(List<Message> messageList) {
        mMessageList = messageList;
    }

}

File: feedback/src/main/java/workshop1024/com/xproject/feedback/model/message/MessageGroup.kt
package workshop1024.com.xproject.feedback.model.message

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "messagegroups")
data class MessageGroup(
        @PrimaryKey @ColumnInfo(name = "groupId") var mGroupId: String,
        @ColumnInfo(name = "publishData") var mPublishData: String,
        @ColumnInfo(name = "messageList") var mMessageList: List<Message>)
